test/gateway/eip20gateway/helpers/helper.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+const web3 = require('../../../test_lib/web3.js');
+const GatewayHelper = require('../../gateway/helpers/helper');
+
+const EIP20GatewayHelper = function(gateway) {
+  const oThis = this;
```

Comment:
Again I don't think you need this here as you don't use `this` in callbacks or similar.